coderd/database/migrations/000070_deployment_graph.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE workspace_agents
+    ADD COLUMN last_connected_replica_id uuid;
```

Comment:
I'd prefer to make this not null with a default `'00000000-0000-0000-0000-000000000000'::uuid` since in practice this should never be null.

Reply:
It can be null if the agent hasn't connected yet

Reply:
oh good point